Client/Components/ExternalServicesComponent.razor.css

```diff
@@ -0,0 +1,28 @@
+﻿button {
+    background-color: #ffffff;
+    border: none;
+    border-radius: 50px;
+    color: #878787;
+    font-family: 'Roboto', sans-serif;
+    font-size: 20px;
+    font-weight: bold;
+    height: 67px;
+    margin-bottom: 1.6rem;
+    outline: none;
+    width: 401px;
+}
+
+    button img {
```

Comment:
wrong code formating

Reply:
In my opionion this formating is correct. It should be tabbed right because it makes code more redeable. Because you modify img inside button so its tabbed right.